buildSrc/src/main/groovy/org.apache.groovy-all.gradle

```diff
@@ -112,4 +112,21 @@ configurations {
             }
         }
     }
+    // GROOVY-10543 - The platform configurations are published as libraries
+    apiElements {
+        attributes {
+            attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category, Category.LIBRARY))
```

Comment:
ihmo java version is missing & required

Reply:
I thought about it and decided to not include it in the first version. Since `groovy-all` does not have a JAR itself, constraining the Java version here is not really needed.
The libraries it depends on do have that attribute and will be the ones failing to resolve if they are consumed in the wrong context.

Reply:
I agree that the Java version doesn't make sense in this context.